db/db_iter.cc

```diff
@@ -107,11 +107,11 @@ Status DBIter::GetProperty(std::string prop_name, std::string* prop) {
 }
 
 bool DBIter::ParseKey(ParsedInternalKey* ikey) {
-  if (ParseInternalKey(iter_.key(), ikey) != Status::OK()) {
-    status_ = Status::Corruption("corrupted internal key in DBIter");
+  Status s = ParseInternalKey(iter_.key(), ikey, false);  // TODO
```

Comment:
Usually when there's a literal value in an argument list I try to comment the name to help out the reader, e.g., `ParseInternalKey(..., false /* log_err_key */);`

Reply:
fixed